### ISSUE TITLE
Documentation around `INSTALLER_NO_MODIFY_PATH` in "standalone installer" section

### ISSUE THREAD

**Issue by rcastellotti** [astral-sh/uv#9835]
Hello 🐻 

TLDR: Should the "opt out" env var be better documented in the Installation step?

I recently started using `uv` and I am now adding it to a Jenkins pipeline.
As of now I am a bit limited and I need to install it in the pipeline itself, so I added the following `script` block:

```groovy
stage("install uv") {
  steps {
    dir("e2e") {
      script {
        sh 'curl -LsSf https://astral.sh/uv/0.5.7/install.sh | env UV_INSTALL_DIR=. sh'
      }
    }
  }
}
```

Something happened on the Jenkins executor making the entire node crash (bash was trying to source a file that does not exist causing some issues).

Only after the facto I realized the script modifies `~/.bashrc`.

```bash
rc@den ~> cat ~/.bashrc
export PATH="$PATH:/Applications/Docker.app/Contents/Resources/bin/"

. "$HOME/.cargo/env"

[[ -f ~/.bash-preexec.sh ]] && source ~/.bash-preexec.sh
eval "$(atuin init bash)"
```

```bash
rc@den ~> curl -LsSf https://astral.sh/uv/0.5.7/install.sh | env UV_INSTALL_DIR=. sh
downloading uv 0.5.7 aarch64-apple-darwin
no checksums to verify
installing to .
  uv
  uvx
everything's installed!
rc@den ~> cat ~/.bashrc
export PATH="$PATH:/Applications/Docker.app/Contents/Resources/bin/"

. "$HOME/.cargo/env"

[[ -f ~/.bash-preexec.sh ]] && source ~/.bash-preexec.sh
eval "$(atuin init bash)"


. "./env"
```
Note:

```diff
--- pre-uv-install-bashrc	2024-12-12 12:36:51
+++ post-uv-install-bashrc	2024-12-12 12:36:53
@@ -6,3 +6,5 @@
 [[ -f ~/.bash-preexec.sh ]] && source ~/.bash-preexec.sh
 eval "$(atuin init bash)"
 
+
+. "./env"
\ No newline at end of file
```

After reading the installer script I noticed the env var `INSTALLER_NO_MODIFY_PATH=1` can be set to prevent this behaviour.
This is documented in the [upgrading-uv](https://docs.astral.sh/uv/getting-started/installation/#upgrading-uv) section, but not in the install section.

I understand automatically populating the `~/.bashrc` file choice by default and optionally opting out of this functionality, but: should this be better documented in the [standalone-installer](https://docs.astral.sh/uv/getting-started/installation/#standalone-installer) section?

P.S: I can open a small PR to update [installation.md](https://github.com/astral-sh/uv/blob/main/docs/getting-started/installation.md), should you think this would be helpful.

P.P.S: Thanks for singlehandedly making the python ecosystem a bit better :)

**Comment by charliermarsh** [comment]
I thin it's intentional that this is documented in https://docs.astral.sh/uv/configuration/installer/ which is linked in `installation.md` under "See the documentation on [installer configuration](https://github.com/astral-sh/uv/blob/main/docs/configuration/installer.md) for details on customizing your uv installation.".

**Comment by rcastellotti** [comment]
Uhm, yea I missed it 🤦 .
In my defense I can only say the "Tip" block is much easier to spot :)
I guess I can close this.
